workers/unity/Assets/Tests/EditmodeTests/Correctness/Subscriptions/CommandTests.cs

```diff
@@ -57,8 +109,8 @@ private static TestCommands.Test.ReceivedResponse ResponseGenerator(CommandReque
         private static EntityTemplate GetTemplate()
         {
             var template = new EntityTemplate();
-            template.AddComponent(new Position.Snapshot(), "worker");
-            template.AddComponent(new TestCommands.Snapshot(), "worker");
+            template.AddComponent(new Position.Snapshot(), WorkerID);
+            template.AddComponent(new TestCommands.Snapshot(), WorkerID);
```

Comment:
There should be an overload w/o the string that you can use (since the write access isn't used at all in the tests).